packages/aws-lambda/test/integration_test/test_definition.js

```diff
@@ -1595,6 +1595,38 @@ function registerTests(handlerDefinitionPath) {
           ])
         ));
   });
+describe('triggered by aws lambda function url', function () {
```

Comment:
The test looks 👍 👍 

One nitpick: The formatting looks off. I would recommend to configure your IDE/editor so that it automatically runs `prettier` when you save a file.

Something like [this](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) maybe for Visual Studio Code (not sure what you are using).

You can also run `npm run prettier` to format all source code files - that might be a good idea because probably one or two other files that you have touched in this PR also need to be reformatted slightly.